### ISSUE TITLE
Explicitation d'une consigne: connexion des lieux. 

### ISSUE THREAD

**Issue by ajaniak** [Chartes-TNAH/Boatlas#1]
Ella, 

je reprends ici la conversation mail avec T.Clérice pour que nous gardions trace de sa réponse.
Ajouter des connexions entre les lieux et les gérer, c'est-à-dire " le lieu "Panthéon" est connecté au lieu Paris, car il fait partie du plus grand ensemble. Une relation de type "partie de" ".

**Comment by ajaniak** [comment]
@DUREAULT, j'y pense mais on va devoir ajouter des lieux, on pourrait peut-être balancer dans la gazetteer les fichiers monuments historiques que nous aurons nettoyés?

**Comment by DUREAULT** [comment]
Yes et dans ces cas-là, on pourrait essayer de voir comment récupérer les coordonnées GPS ?

**Comment by ajaniak** [comment]
Les latitudes et les longitudes sont déjà dans le fichier csv monuments historiques.

**Comment by DUREAULT** [comment]
Au temps pour moi, on parle un peu vite à ces heures-là.

**Comment by DUREAULT** [comment]
Pour continuer sur la connexion entre les lieux, si je comprends bien l'idée de "gérer ces connexions" serait de donner la possibilité à l'utilisateur de faire le lien s'il est connecté ?
Dites-moi si je me trompe @PonteIneptique ?

**Comment by PonteIneptique** [comment]
Oui.
On pourrait aussi imaginer la capacité a faire se lien a la creation

**Comment by DUREAULT** [comment]
Oui et/ou, à partir d'une notice existante, créer une notice complète.

**Comment by ajaniak** [comment]
Ah je pensais qu'il fallait faire en sorte que les jointures se gèrent automatiquement et pas qu'elles soient gérées manuellement

**Comment by PonteIneptique** [comment]
Non, cela tiendrait de l'intelligence artificielle ou *a minima* de l'installation de moteurs de recherche externe.
Je vous propose d'éviter cette complexité.

**Comment by DUREAULT** [comment]
Axelle, je retranscris ici nos choix pour les connexions entre les lieux. 
Nous avons choisi de construire une table "Type" pour gérer les relations entre les lieux avec : 
-un ID 
-un nom de type de relation comme instances de table). 
Cette table sera reliée à une table de relations qui fera la correspondance entre l'ID du type de la relation et les ID des lieux que nous cherchons à relier entre eux. 

Voici le choix de nos types repris en partie du modèle Pléiades (et largement simplifiés) : 

part of (historic) = équivalent découpage administratif antérieur
part of (region) = équivalent découpage administratif actuel
part of (physical, topographical)

**Comment by ajaniak** [comment]
@DUREAULT, 

J'ai essayé un premier modèle pour nos class et nos jointures, enfin de ce que j'ai réussi à comprendre des guidelines. Dis-moi si tu as compris la même chose... je l'ai mis sur l'upstream sur la branche typologies-relations.

**Comment by DUREAULT** [comment]
Du coup tu ne travailles pas en fork ?

**Comment by ajaniak** [comment]
si mais j'ai eu un raté ^^"

**Comment by ajaniak** [comment]
mais heureusement je travaillais sur une branche donc on pire on pourra supprimer!

**Comment by DUREAULT** [comment]
D'accord. Donc du coup je m'occupe de la classe bibliographie ?

**Comment by ajaniak** [comment]
Si seulement on en était déjà la ....j'ai juste tenté de concrétiser (très partiellement d'ailleurs) ce dont on a parlé aujourd'hui. Je suis même pas sûre du modèle donc avant de prendre notre envol ça serait bien qu'on soit raccord non?
